django/contrib/humanize/templatetags/humanize.py

```diff
@@ -140,7 +140,9 @@ def intword(value):
     except (TypeError, ValueError):
         return value
 
-    if value < 1000000:
```

Comment:
Check for value less than 1000000

Reply:
It is checking for a value less than `1000000 `